docs/conf.py

```diff
@@ -187,6 +187,7 @@
             url="https://blueskyproject.io",
         )
     ],
+    navigation_with_keys=True
```

Comment:
I suggest this should be False as per https://github.com/pydata/pydata-sphinx-theme/issues/1492.

Also, we should put this fix into skeleton too

Reply:
done, will make an issue in skeleton(s)